### PR TITLE
Add plugin methods

### PR DESCRIPTION
**Pull request by raphrel**

### CODE REVIEW

File: Sources/Core/Components/PlayerPlugin.swift [hunk 1]
```diff
@@ -27,17 +27,38 @@
...
     func didInit(player: AVPlayer)
     
-    /**
-    Method called when Player is in Loaded state
-     - parameters:
-        - media: PlayerMedia just loaded
-     */
-    func didLoadMedia(_ media: PlayerMedia?)
+    /// Called when Player init Loading state
+    func startLoading()
```
Comment (JiceDessaint) [inline Sources/Core/Components/PlayerPlugin.swift]: You should keep methods name consistency: 
- didStartLoading
- didStartBuffering
...

File: Sources/Core/Components/PlayerPlugin.swift [hunk 2]
```diff
@@ -27,17 +27,38 @@
...
+    
+    /// Called when Player init Buffering state
```
Comment (JiceDessaint) [inline Sources/Core/Components/PlayerPlugin.swift]: I believe "when player *enters* Buffering state" would be better, don't you think?

File: Sources/Core/Components/PlayerPlugin.swift (hunk 1 above)
Comment (raphrel) [reply Sources/Core/Components/PlayerPlugin.swift]: right!

File: Sources/Core/Components/PlayerPlugin.swift (hunk 2 above)
Comment (raphrel) [reply Sources/Core/Components/PlayerPlugin.swift]: yes

**Pull request merged**